bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -460,4 +475,13 @@ private void postFailureEvent(String extensionId, @Nullable String msg) {
         Event event = AddonEventFactory.createAddonFailureEvent(extensionId, msg);
         eventPublisher.post(event);
     }
+
+    private boolean remoteEnabled() {
+        try {
+            Configuration configuration = configurationAdmin.getConfiguration("org.openhab.addons", null);
+            return (boolean) Objects.requireNonNullElse(configuration.getProperties().get("remote"), true);
```

Comment:
It looks.like the [getProperties()](https://docs.osgi.org/javadoc/osgi.cmpn/7.0.0/org/osgi/service/cm/Configuration.html#getProperties--) can return `null` and we are potentially run into NPEs here.

Reply:
Fixed in https://github.com/openhab/openhab-core/pull/2607/commits/fc25ddbf41f4671b165ec795d537e5a6adb1e35a

Reply:
Thank you.

I will try to find a solution for #2581 later.